CheatSheets/java-cheatsheet.md

```diff
@@ -0,0 +1,145 @@
+# Table of Content
+
+ - [Data Types](#data-types)
+ - [Data Conversion](#data-conversion)
+ - [Operators](#operators)
+ - [Statements](#statements)
+ - [String Methods](#string-methods)
+ 
+## Data Types
+| Data Type | Size |
+|--|--|
+| boolean | 1 bit |
+| char | 2 byte |
+| int | 4 byte |
+| short | 2 byte| 
+|long |8 byte |
+|float|4 byte|
+|double|8 byte |
+
+## Data Conversion
+
+### String to Number
+```java
+    int i = Intege­r.p­ars­eInt(_str_);  
+	double d = Double.pa­rse­Double(_str_);
+```
+	
+### Any Type to String
+```java
+	String s = String.va­lueOf(_value_);  
+
+```
+  
+### Numeric Conver­sions
+```java
+	int i = (int) _numeric expression_; 
+
+```
```

Comment:
Is it possible to give them in a `tabular format`? If it is `possible`, then `modify` your code, please.